packages/jellyfish-api-core/src/category/blockchain.ts

```diff
@@ -214,6 +214,16 @@ export class Blockchain {
   async waitForBlockHeight (height: number, timeout: number = 30000): Promise<WaitBlockResult> {
     return await this.client.call('waitforblockheight', [height, timeout], 'number')
   }
+
+  /**
+   * Get statistics about the total number and rate of transactions in the chain.
+   *
+   * @param {number} nBlocks - size of the window in number of blocks
+   * @param {string} blockHash - the hash of the block that ends the window
```

Comment:
we're referring jsdoc syntax (note: not google closure)
https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html

```suggestion
   * @param {number} [nBlocks] - size of the window in number of blocks
   * @param {string} [blockHash] - the hash of the block that ends the window
```

Reply:
better to have default info as well
https://github.com/DeFiCh/ain/blob/612bc1a52f7ab9239767f52d0ac5bd5928979e35/src/rpc/blockchain.cpp#L1623-L1624